src/exampleTypes/TagsQuery/index.js

```diff
@@ -67,7 +67,13 @@ let TagsQuery = ({
               tag => ({ [tagValueField]: tag, distance: 3 }),
               tags
             )
-            tree.mutate(node.path, { tags: [...node.tags, ...tagObjects] })
+            let allTags = [...node.tags, ...tagObjects]
```

Comment:
Nit: if this was named `tags`, you could use object shorthand in the mutate call below

Reply:
done